src/main/java/net/mcreator/element/GeneratableElement.java

```diff
@@ -137,6 +139,7 @@ public GSONAdapter(@Nonnull Workspace workspace) {
 		public GeneratableElement deserialize(JsonElement jsonElement, Type type,
 				JsonDeserializationContext jsonDeserializationContext) throws JsonParseException {
 			ModElement lastModElement = workspace.getModElementManager().getLastElementInConversion();
+			MCREvent.event(new GElementDeserializedEvent(workspace, jsonElement, jsonDeserializationContext));

```

Comment:
Add one new line above to visually separate event from other logic

Reply:
Alright, should I also move this to the part where the it checks if converters exist for the GE? So the logic before that can be executed before calling the event.